docs/source/conf.py

```diff
@@ -2912,6 +2912,7 @@
     "size_bytes",
     # torch.fx.passes.infra.partitioner
     "CapabilityBasedPartitioner",
+    "DependencyViewer",
```

Comment:
Adding new entries to this file is not ok.
You need to either properly document this class on the website by adding it to the appropriate .rst or make it a private API.

Reply:
Thanks.